CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 <!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
 <!-- Please also include if this is a Bug Fix, Enhancement, or Feature -->
 
+* Add organization scope field for oauth clients by @Netra2104 [#812](https://github.com/hashicorp/go-tfe/pull/812)
```

Comment:
💅 You want to make sure your changelog entry is under a specific category: Bug Fixes, Enhancements, Features. You can add them as necessary, example:
```suggestion
## Enhancements
* Add organization scope field for oauth clients by @Netra2104 [#812](https://github.com/hashicorp/go-tfe/pull/812)
```